### ISSUE TITLE
Road Network Improvements in Belarus

### ISSUE THREAD

**Issue by awisemanapple** [osmlab/appledata#17]
# Task Description

This task outlines work to improve the road network in Belarus.

Improvements that will be addressed include the following:

- Correcting road connectivity issues
- Improving road classification consistency according to [global](https://wiki.openstreetmap.org/wiki/Key:highway) and [local](https://wiki.openstreetmap.org/wiki/RU:WikiProject_Belarus) policy
- Adding missing roads
- Correcting road alignment issues

## Mapping Guidelines

The team will follow [OSM](http://wiki.openstreetmap.org/wiki/Highways) and [local](https://wiki.openstreetmap.org/wiki/RU:WikiProject_Belarus) policy, along with any other guidelines as appropriate. 

The team will use the imagery and data sources specified below for mapping in Belarus.

When in doubt, the team will use the existing tags and data which are used locally. The history in JOSM (ctrl-H) identifies what previous mappers added and their notes and/or explanations regarding changes made. This history can also show if a previous mapper has visited that area or has expert knowledge or sources.

## Area of Focus

Road network throughout the country.  

## Tools

The team will use [JOSM](https://josm.openstreetmap.de) for completing the task. JOSM has validation warnings for road networks and other possible issues that are important to prevent improper changes. 

## Sources

The team will use the imagery and data sources as available including: 
- DigitalGlobe Standard/Premium
- Bing imagery
- Mapbox imagery
- ESRI imagery
- HOT task imagery, if available
- Mapillary
- OpenStreetCam
- Pic4Carto  
- Telenav's MissingRoads plugin and [ImproveOSM](https://improveosm.org/#background=Bing&map=2.43/53.19/72.35)
- Strava and OSM GPS traces

The team will apply image offsets as needed.

## Changeset Comments
The team will provide changeset comments that are in compliance with [OSM changeset guidelines](http://wiki.openstreetmap.org/wiki/Good_changeset_comments). 

## Error Detection

The team will check for errors visually and using [JOSM validation warnings](http://wiki.openstreetmap.org/wiki/JOSM/Validator) prior to committing the changeset.

In addition, the team will review its work in accordance with validation guidelines such as the [OSM Wiki](http://wiki.openstreetmap.org/wiki/OSM_Tasking_Manager/Validating_data) and [LearnOSM](http://learnosm.org/en/coordination/review/).

## Contributors

- [Data Team](https://github.com/osmlab/appledata/wiki/Data-Team)

## For more information
Please contact our community projects lead [Andrew Wiseman](https://www.openstreetmap.org/user/Marion%20Barry)

**Comment by Komzpa** [comment]
For imagery, some spots in Minsk and around are on OpenAerialMap.

**Comment by tohaklim** [comment]
Mapillary has some coverage of the country as well, as you might've seen (not all of it useful though).
The main issues are with OSM data itself being incomplete (hence rendering your import incomplete). Biggest gaps would be newer neighbourhoods/suburbs (usually already mapped in large cities) and rural areas (usually either crudely traced, not mapped or, being erroneously classified, not imported/updated. 
PS: Some of the Apple Maps imagery (certainly in Hrodna) is quite fresh (July-August, DigitalGlobe?), but unfortunately is probably copyrighted.

**Comment by awisemanapple** [comment]
Hi @Komzpa @tohaklim, thanks for the suggestions! That's very helpful.

**Comment by EUSkyDive** [comment]
@awisemanapple, are you planning to add/update MapRoulette with new challenges for Belarus?

**Comment by awisemanapple** [comment]
@EUSkyDive thanks for the request! Sure, let me get started on that. It might take a little bit of time.

**Comment by BelarusianNerd** [comment]
Maps are getting worse and worse from version to version.

Here is the list of major issues:
1. Building numbers are missing across the country.
2. Duplicated labels for inhabited localities.
3. Incorrect mapping of the Minsk (city) and Minskaja voblasć (region). [Minsk (city)](https://www.openstreetmap.org/relation/59195) is not a part of [Minskaja voblasć (region)](https://www.openstreetmap.org/relation/59752).
4. Labels of these localities are displayed in different languages at the same time. Some of them in Belarusian, some in Russian, some use [proper international names (Rules accepted by the local authorities and the UN)](https://en.wikipedia.org/wiki/Instruction_on_transliteration_of_Belarusian_geographical_names_with_letters_of_Latin_script) and also romanized names from Russian (which is not correct). Based on the local regulations names of the geographical objects are defined in the Belarusian language and Belarusian Latin alphabet (link to the instruction provided earlier).
5. For motorways, highways and roads an incorrect signs design is used.
5. Public transport stops are mostly missing even being available on the OSM.

**Comment by awisemanapple** [comment]
@BelarusianNerd @tohaklim Thanks for the feedback! On 1., do you mean addresses or something else? For the others, providing feedback from in the Maps app would be helpful so we can route it to the right place. I can do the same thing if you can you provide some examples. Via email might be easier, I am andrew_wiseman@apple.com

**Comment by BelarusianNerd** [comment]
@awisemanapple, yes I meant addresses. As for now address for the building contains almost everything (street, postal/ZIP code, city/town/village, region and country) except the number.
Sure, I or another person we discussed these issues with will send you more detailed information. We tried to reach Apple Maps team via the Feedback Assistant and In-App reporting, but unfortunately, haven't seen any progress or feedback.
